src/controllers/authController.ts

```diff
@@ -1,49 +1,67 @@
 import { Request, Response } from 'express';
-import { User } from '../entities/User';;
+import { User } from '../entities/User';
 import bcrypt from 'bcrypt';
 import { getRepository } from 'typeorm';
+import sendMail from '../utils/sendMail';
+import { send } from 'process';
+import dotenv from 'dotenv';
+dotenv.config();
 
+const registerUser = async (req: Request, res: Response) => {
```

Comment:
Use `export const registerUser`
You won't need to export like this `export   {registerUser }`